avatar/__init__.py

```diff
@@ -206,6 +207,48 @@ def normalize(a: Any) -> Any:
     return wrapper
 
 
+def enableFlag(flag):
+    """ Enable aconfig flag.
+    Requires that the test class declares a devices: Optional[PandoraDevices] attribute.
```

Comment:
I think an empty line might be required between the short description and the detailed one.

Reply:
Done!